.github/workflows/update_changelog.yml

```diff
@@ -0,0 +1,57 @@
+name: Update Changelog.md
+
+on:
+  workflow_run:
+    workflows: ["Check PR for Changelog"]
```

Comment:
I don't think we need this trigger here? Only need to react to the pull_request closed event below. This trigger will never pass the `github.event.pull_request.merged` check (and may even fail as it won't have a pull_request attribute).

Reply:
Hi @rtibbles. I understood your concern regarding taking the target branch into consideration for checkout.
I'll make the required changes within some days time.